src/Tasks/Common/ConflictResolution/ResolvePackageFileConflicts.cs

```diff
@@ -44,6 +55,7 @@ protected override void ExecuteCore()
         {
             var log = new MSBuildLog(Log);
             var packageRanks = new PackageRank(PreferredPackages);
+            var packageOverrides = new PackageOverrideResolver<ConflictItem>(PackageOverrides);
```

Comment:
How much perf did you give back by creating this every time the task runs? I thought the version with a hard coded structure created every time was significantly slower than a static list. 

The parsing logic looks very heavy on allocations: split, trim, tuple, yield.

Reply:
My original mistake was doing the data creation in the `ConflictResolver` constructor, which is called 3x per invocation of the task.  So that's why it came up when I profiled at that time (it was 3x what it needed to be).

I did some profiling of my current approach, and this creation takes `<10%` of the time of the new task. The real time left in the task (according to PerfView) is getting Item Metadata multiple times for every ConflictItem.  This accounts for `~60%` of the time of the new task according to the profiles I've looked at.

Reply:
BTW - I've updated the original post with the perf numbers I'm seeing on my machine.

Reply:
OK, I'm satisfied. We can tune the rest if it ever it shows up and this is clearly a nice win already.

Reply:
You made my realize that I should be lazy-loading this data for the case where there are no conflicts.  No reason to build the dictionary up, until we need it.  I've pushed a change for that.